polytope/polytope.py

```diff
@@ -204,14 +204,25 @@ def __copy__(self):
     def __contains__(self, point, abs_tol=ABS_TOL):
         """Return True if polytope contains point.
 
+        @type point: tuple, 1d array, or 2d array (of column vectors)
+
+        @rtype: bool, 1d array
+
         See Also
         ========
         L{is_inside}
         """
         if not isinstance(point, np.ndarray):
             point = np.array(point)
-        test = self.A.dot(point.flatten()) - self.b < abs_tol
-        return np.all(test)
+
+        if point.ndim == 1:
+            point.shape = (point.size, 1)
```

Comment:
Yes, that, but also it converts a 1D array to a 2D array. if the point is only 1D, then the matrix multiplication a few lines down throws an error.